### ISSUE TITLE
"Editado por" not displaying in Spanish lessons 

### ISSUE THREAD

**Issue by mariajoafana** [programminghistorian/jekyll#517]
Is missing from all the Python series and from the following lessons:
[http://programminghistorian.org/es/lecciones/introduccion-control-versiones-github-desktop](url)
[http://programminghistorian.org/es/lecciones/sparql-datos-abiertos-enlazados](url)
[http://programminghistorian.org/es/lecciones/creando-diagramas-de-redes-desde-fuentes-historicas](url)
[http://programminghistorian.org/es/lecciones/preservar-datos-de-investigacion](url)
[http://programminghistorian.org/es/lecciones/escritura-sostenible-usando-pandoc-y-markdown](url)

**Comment by wcaleb** [comment]
This raises a larger question. Should the "editor" for the English lesson also be displayed as the editor for the Spanish translation? Is there a "Spanish editor" role that needs to be distinguished from "original editor"? I though the only roles unique to the Spanish lessons were "translator" and "translator reviewer." So, for example, on "creando-diagramas-de-redes...," wouldn't the editor still be Fred? 

Apologies if this was discussed elsewhere and I missed it.

**Comment by wcaleb** [comment]
P.S. The technical issue causing the problem is that the yaml field should be "editors," not "editor". Perhaps my above question could be resolved if the Spanish editor is listed together with the original English editor as "editors."

**Comment by acrymble** [comment]
I thought she meant in the author listing
![screen shot 2017-07-14 at 16 08 29](https://user-images.githubusercontent.com/646647/28217906-be8a336a-68ae-11e7-8450-057b7ffc902c.png)

**Comment by acrymble** [comment]
Oh nevermind, I see you'er right @wcaleb . Sorry.

**Comment by fredgibbs** [comment]
in many cases, there is no editor or editors yaml field listed at all. i think the best solution is actually to make all yaml fields plural (editors, reviewers, translators, translation-reviewers) to minimize future confusion. this would mean updating yaml in all the spanish lessons to have translators and translation-reviewers (the other fields are already plural on all lessons) and update the lessons template to use the plural fields. this is obviously more work than just adding the editors field or changing editor to editors in the spanish lessons, but i think it's worth it for consistency and avoiding this problem in the future (a very easy mistake to make).

**Comment by mdlincoln** [comment]
If we do set up the Spanish lesson to display both the translation editor as well as the original editor, I would suggest splitting apart an `editors` and `translation-editors` field in the YAML. Even if we'd like to display both editors in equal positions on the lesson page, I want to make sure we can keep that knowledge structured distinctly, in case we later think of some reason we want to distinguish the editors.

**Comment by mdlincoln** [comment]
OK let's fix one problem at a time. I'm creating a separate issue about plural fields.

For this issue: @mariajoafana @vgayolrs @arojascastro 

Please add `editor` fields for editor of the original lesson, and `translation-editors` field with the translation editor. 

Once this is done, we can configure display of both fields. We can also then do a find/replace later if we really want to switch all these to plural fields.

**Comment by mariajoafana** [comment]
Ok, thanks, on it.

**Comment by mdlincoln** [comment]
@mariajoafana the following lessons need `editors` added

* In /es/lecciones/contar-frecuencias.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/crear-y-ver-archivos-html-con-python.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/de-html-a-lista-de-palabras-1.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/de-html-a-lista-de-palabras-2.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/escritura-sostenible-usando-pandoc-y-markdown.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/instalacion-linux.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/instalacion-mac.md:

  - [ ] 'editors' is missing.
* In /es/lecciones/instalacion-windows.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/instalar-modulos-python-pip.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/introduccion-a-bash.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/introduccion-control-versiones-github-desktop.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/introduccion-e-instalacion.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/manipular-cadenas-de-caracteres-en-python.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/normalizar-datos.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/palabras-clave-en-contexto-n-grams.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/preservar-datos-de-investigacion.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/reutilizacion-de-codigo-y-modularidad.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/salida-de-datos-como-archivo-html.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/salida-palabras-clave-contexto-ngrams.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/sparql-datos-abiertos-enlazados.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/trabajar-con-archivos-de-texto.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/trabajar-con-paginas-web.md:
  - [ ] 'editors' is missing.
* In /es/lecciones/ver-archivos-html.md:
  - [ ] 'editors' is missing.

**Comment by vgayolrs** [comment]
Hi @mdlincoln , allmost done. In [Installing Python Modules with pip](https://programminghistorian.org/lessons/installing-python-modules-pip) -Spanish version "/es/lecciones/instalar-modulos-python-pip.md"-, there is no editor.

**Comment by mdlincoln** [comment]
Thanks for fixing these @vgayolrs ! The original lesson never had an editor, actually - so I added a special variable to the YAML that will keep the missing editor from causing an error.
